full/Dockerfile

```diff
@@ -13,3 +13,6 @@ ENV PATH=${CARGO_HOME}/bin:${PATH}
 
 # Adding ARMV6M target to the default toolchain
 RUN rustup target add thumbv6m-none-eabi
+
+# Adding rust-src component
+RUN rustup component add rust-src
```

Comment:
Hello @carlosala, could you please explain why this component is required ?

Reply:
It's used to compile apps that have some parts of Rust on them. Compiling from source rust-std reduces the binary size dramatically.
This addition needs to be added in order to be able to run the static analyzer, and all the stuff that runs in ledger reusable workflow.

Reply:
It should not affect at all the container, just having rust std source code available for its usage.